src/Compilers/CSharp/Test/Syntax/Parsing/RadOnlyStructs.cs

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Xunit;
+using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit.Abstractions;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Parsing
+{
+    [CompilerTrait(CompilerFeature.ReadonlyReferences)]
```

Comment:
The OCD in me can't help but mention the casing inconsistency here:

- `CompilerFeature.ReadonlyReferences`
- `class ReadOnlyStructs`

Prefer consistent casing here around `ReadOnly`.